src/pages/home/report/ParticipantLocalTime.js

```diff
@@ -40,7 +41,7 @@ class ParticipantLocalTime extends PureComponent {
     }
 
     getParticipantLocalTime() {
-        const reportRecipientTimezone = lodashGet(this.props.participant, 'timezone', {});
+        const reportRecipientTimezone = lodashGet(this.props.participant, 'timezone', CONST.DEFAULT_TIME_ZONE);
```

Comment:
setting a proper default because if we use `{}` as default then it will fail 3 lines down for `currentUserDay` and throw an error.